src/plugins/plot/src/inspector/PlotOptionsController.js

```diff
@@ -23,13 +23,11 @@
 define([
     '../configuration/configStore',
     '../lib/eventHelpers',
-    '../../../../api/objects/object-utils',
-    'lodash'
+    '../../../../api/objects/object-utils'
 ], function (
     configStore,
     eventHelpers,
-    objectUtils,
-    _
+    objectUtils
```

Comment:
I dont see this used anywhere

Reply:
Removed and also aliased objectUtils and shortened all the import paths to it.